version.go

```diff
@@ -12,10 +12,21 @@ import (
 	"strings"
 )
 
-// Commit stores the current commit hash of this build, this should be set
-// using the -ldflags during compilation.
+// Commit stores the current git tag of this build, when the build is based on
+// a tagged commit. If the build is based on an untagged commit or is a dirty
+// build, the Commit field stores the most recent tag suffixed by the commit
+// hash, and/or "-dirty". This should be set using the -ldflags during
+// compilation.
 var Commit string
 
+// CommitHash stores the current git commit hash of this build. This should be
+// set using the -ldflags during compilation.
+var CommitHash string
+
+// Dirty stores a "-dirty" string, if the build had uncommitted changes when
```

Comment:
... so then here it'd be `dirty` too (and where we concatenate we add the dash).